packages/calypso-products/src/constants/akismet.ts

```diff
@@ -104,16 +161,21 @@ export const AKISMET_UPGRADES_PRODUCTS_MAP = {
 	[ PRODUCT_AKISMET_PERSONAL_YEARLY ]: `/checkout/akismet/${ PRODUCT_AKISMET_PLUS_YEARLY }`,
 	[ PRODUCT_AKISMET_PLUS_MONTHLY ]: `/checkout/akismet/${ PRODUCT_AKISMET_PLUS_20K_MONTHLY }`,
 	[ PRODUCT_AKISMET_PLUS_YEARLY ]: `/checkout/akismet/${ PRODUCT_AKISMET_PLUS_20K_YEARLY }`,
+	[ PRODUCT_AKISMET_PLUS_BI_YEARLY ]: `/checkout/akismet/${ PRODUCT_AKISMET_PLUS_20K_BI_YEARLY }`,
 	[ PRODUCT_AKISMET_PLUS_20K_MONTHLY ]: `/checkout/akismet/${ PRODUCT_AKISMET_PLUS_30K_MONTHLY }`,
 	[ PRODUCT_AKISMET_PLUS_20K_YEARLY ]: `/checkout/akismet/${ PRODUCT_AKISMET_PLUS_30K_YEARLY }`,
+	[ PRODUCT_AKISMET_PLUS_20K_BI_YEARLY ]: `/checkout/akismet/${ PRODUCT_AKISMET_PLUS_30K_BI_YEARLY }`,
 	[ PRODUCT_AKISMET_PLUS_30K_MONTHLY ]: `/checkout/akismet/${ PRODUCT_AKISMET_PLUS_40K_MONTHLY }`,
 	[ PRODUCT_AKISMET_PLUS_30K_YEARLY ]: `/checkout/akismet/${ PRODUCT_AKISMET_PLUS_40K_YEARLY }`,
+	[ PRODUCT_AKISMET_PLUS_30K_BI_YEARLY ]: `/checkout/akismet/${ PRODUCT_AKISMET_PLUS_40K_BI_YEARLY }`,
 	[ PRODUCT_AKISMET_PLUS_40K_MONTHLY ]: `/checkout/akismet/${ PRODUCT_AKISMET_ENTERPRISE_MONTHLY }`,
 	[ PRODUCT_AKISMET_PLUS_40K_YEARLY ]: `/checkout/akismet/${ PRODUCT_AKISMET_ENTERPRISE_YEARLY }`,
-	[ PRODUCT_AKISMET_ENTERPRISE_MONTHLY ]: 'https://akismet.com/enterprise',
-	[ PRODUCT_AKISMET_ENTERPRISE_YEARLY ]: 'https://akismet.com/enterprise',
-	[ PRODUCT_AKISMET_ENTERPRISE_350K_MONTHLY ]: 'https://akismet.com/enterprise',
-	[ PRODUCT_AKISMET_ENTERPRISE_350K_YEARLY ]: 'https://akismet.com/enterprise',
+	[ PRODUCT_AKISMET_PLUS_40K_BI_YEARLY ]: `/checkout/akismet/${ PRODUCT_AKISMET_ENTERPRISE_BI_YEARLY }`,
+	[ PRODUCT_AKISMET_ENTERPRISE_MONTHLY ]: `/checkout/akismet/${ PRODUCT_AKISMET_ENTERPRISE_350K_MONTHLY }`,
+	[ PRODUCT_AKISMET_ENTERPRISE_YEARLY ]: `/checkout/akismet/${ PRODUCT_AKISMET_ENTERPRISE_350K_YEARLY }`,
+	[ PRODUCT_AKISMET_ENTERPRISE_BI_YEARLY ]: `/checkout/akismet/${ PRODUCT_AKISMET_ENTERPRISE_350K_YEARLY }`,
```

Comment:
This is the only instance of a bi-yearly plan "upgrading" to a yearly plan. Since Akismet plans are able to be purchased concurrently and they are not automatically replaced on the backend we shouldn't have any technical issues with this. If we ever incorporate 2 year plans for the higher enterprise plans, this should be updated to upgrade to the 350K bi-yearly plan instead